### DISCUSSION TITLE
When enabling my code, the console doesnt return anything, thus not injecting my code into the bot.

### DISCUSSION THREAD

**Discussion by RevientaChotas** [eternnoir/pyTelegramBotAPI#1929] [Q&A]
Ive tried many tutorials, although none made a difference. 

import telebot

bot_token = '6233021641:AAEJl1Bv6FqU4wfxb4p-BV9WQShdlTLPpbU'

bot = telebot.TeleBot(token=bot_token)

@bot.message_handler(content_types=['text'])
def get_text_messages(message):
    if message.text == "Привет":
        bot.send_message(message.from_user.id, "Привет.")
    elif message.text == "/help":
        bot.send_message(message.from_user.id, "Напиши Привет")
    else:
        bot.send_message(message.from_user.id, "Я тебя не понимаю. Напиши /help.")

im using a virtual environtment setting for this code.

thanks in advance.
- Ilya.

**Comment by Badiboy** [comment]
Just start here.

https://github.com/eternnoir/pyTelegramBotAPI/blob/master/examples/echo_bot.py

**Reply by RevientaChotas** [reply]
nope, still not working. Thanks for the help though!

**Reply by Badiboy** [reply]
Yep, it **definitely works**. For all 453K users per month.
